### PR TITLE
[zh-cn] fix some wording in connect-applications-service.md

### PR DESCRIPTION
**Pull request by yulng**
[zh-cn] fix some wording in connect-applications-service.md
thanks

### CODE REVIEW

File: content/zh-cn/docs/tutorials/services/connect-applications-service.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ the first step:
 Service Selector 将持续评估，结果被 POST
 到使用{{< glossary_tooltip text="标签" term_id="label" >}}与该 Service 连接的一个 EndpointSlice。
 当 Pod 终止后，它会自动从包含该 Pod 的 EndpointSlices 中移除。
-新的能够匹配上 Service Selector 的 Pod 将自动地被为该 Service 添加到 EndpointSlice 中。
+将自动添加与 Service Selector 匹配的新Pod到该 Service 的 EndpointSlice 中。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/services/connect-applications-service.md:201]: 这样改动之后，句子变成了主动句，但是缺少主语，反倒不如原来明确。
原来的句子不通顺，可以考虑改成 “... 将被自动替该 Service 添加到 ...”

Comment (yulng) [reply content/zh-cn/docs/tutorials/services/connect-applications-service.md]: ok, 已改好

**Review by windsonsea — COMMENTED** [review]
Checked the [zh-cn history](https://github.com/kubernetes/website/commits/main/content/zh-cn/docs/tutorials/services/connect-applications-service.md), no new revision of the en upstream after the last sync.

/approve

**Pull request merged**